MDSassignment/guestbook/post/models.py

```diff
@@ -0,0 +1,23 @@
+from django.db import models
+from django.utils import timezone
+
+
+STATUS_CHOICES = (
+	('h', 'Hidden'),
```

Comment:
The way that we'd normally do this is to use variables for the values so that we can refer to them elsewhere. The `'h'` here would be considered a magic value (bad).

I'll make a quick example for you!